<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyMeasureHolePosFormClass</class>
 <widget class="GUIFormBase" name="PropertyMeasureHolePosFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertyMeasureHolePosForm</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetBlockInf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ライブラリ名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ブロック総数</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>使用されている穴検査領域</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分類選択</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyMeasureHolePos.qrc">
     <normaloff>:/Qtres/Class.png</normaloff>:/Qtres/Class.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelLibFolderName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ＸＸＸ</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QToolButton" name="toolButtonArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>描画領域</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyMeasureHolePos.qrc">
      <normaloff>:/Qtres/draw_pen_64.png</normaloff>:/Qtres/draw_pen_64.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonHole">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>穴領域</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyMeasureHolePos.qrc">
      <normaloff>:/Qtres/Hole.png</normaloff>:/Qtres/Hole.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonEdge">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>探索領域</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyMeasureHolePos.qrc">
      <normaloff>:/Qtres/Search.png</normaloff>:/Qtres/Search.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibrary">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Library</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyMeasureHolePos.qrc">
     <normaloff>:/Qtres/Edit.png</normaloff>:/Qtres/Edit.png</iconset>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>穴リスト</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetHoleList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>401</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Page</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Search</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>詳細情報</string>
    </attribute>
    <widget class="QFrame" name="frameBlockInfoOnMouse">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>401</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>マウスカーソル位置の穴検査情報</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>351</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="EditLibName">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Library Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="EditLibID">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>12345</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>名称</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxPosPrecision">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>位置精度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>ノイズサイズ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_92">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>探索ドット</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>穴径精度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxNoiseSize">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="EditSearchDot">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>140</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDiaPrecision">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>311</width>
         <height>171</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="Color">
        <widget class="QFrame" name="frameColorCube">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>0</y>
           <width>231</width>
           <height>131</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonNColor">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>51</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonCreateNew">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>140</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Create new</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonUpdate">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>140</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Mono">
        <widget class="QLabel" name="label_93">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>抽出輝度　暗側</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QSpinBox" name="EditPickupH">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>30</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_94">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>10</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>抽出輝度　明側</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QSpinBox" name="EditPickupL">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonUpdateMono">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>70</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
         <property name="icon">
          <iconset resource="PropertyMeasureHolePos.qrc">
           <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonCreateNewMono">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Create new</string>
         </property>
         <property name="icon">
          <iconset resource="PropertyMeasureHolePos.qrc">
           <normaloff>:/Qtres/Make.png</normaloff>:/Qtres/Make.png</iconset>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QSpinBox" name="spinBoxBandWidth">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>帯幅</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyMeasureHolePos.qrc"/>
 </resources>
 <connections/>
</ui>
